testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -90,8 +90,8 @@ struct EventObserver {
     endpoint: String,
     /// Timeout for sending events to this observer
     timeout: Duration,
-    /// Lossy observers do not retry on error
-    lossy: bool,
+    /// force observers to not retry on error
```

Comment:
Maybe this could be more clear and more scary:
```
/// If true, the stacks-node will not retry if event delivery fails for any reason.
/// WARNING: This should not be set on observers that require successful delivery of all events.
```